modules/apps/layout/layout-content-page-editor-web/src/main/java/com/liferay/layout/content/page/editor/web/internal/display/context/ContentPageEditorDisplayContext.java

```diff
@@ -747,7 +751,29 @@ protected String getResourceURL(String resourceID, boolean doAsGuest)
 	}
 
 	protected long getSegmentsExperienceId() {
-		return SegmentsExperienceConstants.ID_DEFAULT;
+		if (_segmentsExperienceId != null) {
+			return _segmentsExperienceId;
+		}
+
+		Layout layout = themeDisplay.getLayout();
+
+		UnicodeProperties unicodeProperties =
```

Comment:
Hi @cgoncas,

Are we setting these type properties from somewhere else inside liferay-portal?

If this is just meant to be overriden by the module that will be sent to the customer I think it we should add a comment here pointing to the PTR to avoid someone deleting this code in the future:

```
// PTR-2322
```

Apart from that it looks good.

Reply:
This change has been already merged on Liferay: https://github.com/liferay-tango/liferay-portal/pull/1122